src/Panel.js

```diff
@@ -69,13 +75,16 @@ const Panel = React.createClass({
     let collapseClass = this.prefixClass('collapse');
 
     return (
-      <div
-        className={classNames(this.getCollapsibleClassSet(collapseClass))}
-        id={this.props.id}
-        ref='panel'
-        aria-expanded={this.isExpanded() ? 'true' : 'false'}>
-        {this.renderBody()}
-      </div>
+      <Collapse in={this.isExpanded()}>
+        <div
+          className={collapseClass}
+          id={this.props.id}
+          ref='panel'
+          aria-expanded={this.isExpanded() ? 'true' : 'false'}>
```

Comment:
cool.. didn't know that it is possible to write it without commas.
but it seems it is possible only in JSX.